build/scons-install.js

```diff
@@ -38,7 +38,7 @@ function install(versionTag, next) {
 
 	// TODO Combine with unzip method in packager.js?
 	// TODO Support unzipping on windows
-	exec('/usr/bin/unzip -q -o -d "' + dest + '" "' + zipfile + '"', function (err, stdout, stderr) {
+	exec('unzip -q -o -d "' + dest + '" "' + zipfile + '"', function (err, stdout, stderr) {
```

Comment:
`unzip` is not a valid command on Windows. You should use the unzip function in `node-appc`. To do this, first require `node-appc`:

```js
var appc = require('node-appc');
```

Then unzip!

```js
appc.zip.unzip(zipfile, dest, next);
```